Bottom: 457f054b6751cd4a2809429f613715919f9f4c2f
Top:    52e3e5d3ea263fc445efaba8d283fc8a708c84c4
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-17 13:51:30 -0500

Refresh of update-ptxdist-build-factories

---
diff --git a/docker/buildbot-master/buildbot/ptxdist.py b/docker/buildbot-master/buildbot/ptxdist.py
index e50d8c6..a4459b8 100644
--- a/docker/buildbot-master/buildbot/ptxdist.py
+++ b/docker/buildbot-master/buildbot/ptxdist.py
@@ -93,17 +93,6 @@ schedulers = [
         ],
     ),
 
-    schedulers.Nightly(
-        name="ptxdist-nightly",
-        branch=None,
-        builderNames=[
-            "current_armeb_xscale",
-            "current_i686",
-            "current_am335x",
-        ],
-        hour=22
-    ),
-
     schedulers.Triggerable(
         name="upgrade_i686",
         builderNames=["upgrade_i686"]),
